megaparse/api/app.py

```diff
@@ -46,20 +47,20 @@ def _check_free_memory() -> bool:
 @app.post("/v1/file")
 async def parse_file(
     file: UploadFile = File(...),
-    method: ParserType = ParserType.UNSTRUCTURED,
-    strategy: StrategyEnum = StrategyEnum.AUTO,
-    check_table=False,
-    language: Language = Language.ENGLISH,
-    parsing_instruction: str | None = None,
-    model_name: str | None = None,
+    method: ParserType = Form(ParserType.UNSTRUCTURED),
```

Comment:
We should check if the FastAPI enforces type checks on the form struct using pydantic